Apps/Medication/src/Api/IOdrApi.cs

```diff
@@ -29,15 +29,15 @@ public interface IOdrApi
         /// </summary>
         /// <param name="request">The request to query.</param>
         /// <returns>The MedicationHistory.</returns>
-        [Post("/odr/patientProfile")]
```

Comment:
I think this should not be changed, otherwise it would break current proxy usage

Reply:
When the Refit client is added, the appropriate prefix is added depending on the value of the ClientCertificate.Enabled configuration setting.